dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/ClassPathBuilderTest.java

```diff
@@ -81,7 +81,7 @@ public void testArtifactsToPaths_removingDuplicates() throws RepositoryException
   @Test
   public void testBomToPaths_firstElementsAreBomMembers() throws RepositoryException {    
     DefaultArtifact bom =
-        new DefaultArtifact("com.google.cloud:cloud-oss-bom:pom:1.0.0-SNAPSHOT");
+        new DefaultArtifact("com.google.cloud:google-cloud-bom:0.81.0-alpha");
```

Comment:
Now this test is fixed to this BOM. No need to update this test when we upgrade members of Cloud OSS BOM.